README.md

```diff
@@ -18,7 +18,7 @@
 2. **Clone your Repository:** Clone your forked repository URL to your local machine using this command:
 
     ```
-    git clone <your-cloned-repo-url>
+    git clone https://github.com/developer-student-clubs/qUIck-react/
```

Comment:
No, You have to put your **cloned** repo URL there.

Reply:
I cannot do `Step 3 - cd qUIck-react` mentioned in the README.md without cloning this repo first. I think and extensive solution would be to clone this repo and the copy the content of src/components folder to the user's repo's src folder.

Reply:
I am sorry. I had misunderstood.